package.json

```diff
@@ -29,20 +27,21 @@
   },
   "homepage": "https://github.com/batata-frita/lekplats#readme",
   "devDependencies": {
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
+    "react": "^15.4.2 || ^16.0.0",
+    "react-dom": "^15.4.2 || ^16.0.0",
     "sagui": "^8.0.2"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": "^15.4.2 || ^16.0.0",
+    "react-dom": "^15.4.2 || ^16.0.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "html2react": "^1.0.2",
+    "html2react": "^1.1.1",
```

Comment:
We should probably wait for your other PR to https://github.com/Deschtex/html2react/pull/27 to be merged and bumped before merging this one.

Then we will be all supporting React 16.